flight/Modules/OnScreenDisplay/osd_menu.c

```diff
@@ -216,7 +217,7 @@ enum menu_fsm_state {
 	FSM_STATE_STICKLIMITS_ROLLEXH,  /*!< Roll expo horizon */
 	FSM_STATE_STICKLIMITS_PITCHEXH, /*!< Pitch expo horizon */
 	FSM_STATE_STICKLIMITS_YAWEXH,   /*!< Yaw expo horizon */
-	FSM_STATE_STICKLIMITS_ACROINSANITYFACTOR, /*!< Acro insanity factor for AcroPlus */
+	FSM_STATE_STICKLIMITS_MOTORPOWER, /*!< Motor power gain scale */
```

Comment:
I usually read the commit log.  But this is pretty minor IMO and wouldn't make it, because it's not the total removal of the parameter-- just the removal in the OSD config.